digdag-docs/src/operators.rst

```diff
@@ -870,3 +870,81 @@ embulk>: Embulk data transfer
 
   * :command:`embulk>: embulk/mysql_to_csv.yml`
 
+
+s3_wait>: Wait for a file in Amazon S3
+--------------------------------------
+
+The **s3_wait>:** operator waits for file to appear in Amazon S3.
+
+
+.. code-block:: yaml
+
+    +wait:
+      s3_wait>: my-bucket/my-key
+
+Secrets
+~~~~~~~
+
+:command:`aws.access-key`
+  The AWS Access Key ID to use when accessing S3.
+
+:command:`aws.secret-key`
+  The AWS Secret Access Key when accessing S3.
+
+:command:`aws.sse-key`
+  An optional Server-Side Encryption key to use when accessing S3. Must be Base64 encoded.
+
+:command:`aws.sse-key-algorithm`
+  An optional Server-Side Encryption key algorithm to use when accessing S3.
```

Comment:
We'd better add something like `Currently, "AES256" is the only supported algorithm`